change_log/next/FE-1416-simple-color-picker.yml

```
@@ -0,0 +1 @@
+Component Update: "Update component to use styled-components and remove input decorator"
```

Need to be more explicit about which component was updated here:

```suggestion
Component Update: "Simple Colour Picker component has been updated to use styled-components and remove the input decorator"
```